tests/test_inter_area_agent.py

```diff
@@ -399,7 +399,7 @@ def test_iaa_event_trade_buys_accepted_bid(iaa_double_sided):
     assert len(iaa_double_sided.lower_market.calls_energy_bids) == 1
 
     expected_price = 10 * (1 - iaa_double_sided.lower_market.transfer_fee_ratio)
-
+    print(expected_price)
```

Comment:
Please remove

Reply:
There are some other prints in the tests, too.

Reply:
Thanks, removing all prints